static/malawi/config.py

```diff
@@ -259,11 +259,44 @@ class Messages(object):
                 "'help codes' for a list of commodity codes; " + \
                 "'help start' or 'help stop' to start and stop reminders."
 
+    # "rm" keyword
+    # Responses to Facility user reporting the refrigerator malfunction
+    FRIDGE_FACILITY_NO_GAS = "Thank you. The district will be informed that your refrigerator is out of gas."
+    FRIDGE_FACILITY_POWER_FAILURE = "Thank you. The district will be informed that your refrigerator has a power failure."
+    FRIDGE_FACILITY_BREAKDOWN = "Thank you. The district will be informed that your refrigerator is not working."
+
+    # Message to the District user notifying of the refrigerator malfunction
+    FRIDGE_DISTRICT_NO_GAS = "%(facility)s has refrigerator malfunction due to no gas."
+    FRIDGE_DISTRICT_POWER_FAILURE = "%(facility)s has refrigerator malfunction due to power failure."
+    FRIDGE_DISTRICT_BREAKDOWN = "%(facility)s has refrigerator malfunction due to breakdown."
+
+    FRIDGE_HELP = "To report a refrigerator malfunction, please send 'rm [reason code]'"
+    FRIDGE_HELP_REASON = ("'%(code)s' is not a valid reason code. Refrigerator malfunction reason code should be "
+        "1:no gas, 2:power failure, 3:breakdown.")
+
+    FRIDGE_MALFUNCTION_ALREADY_REPORTED = ("You already reported a refrigerator malfunction %(days)s days ago. "
+        "If that malfunction has been fixed, please reply with 'rf', and then report the new malfunction with 'rm'")
+
+    FRIDGE_NOT_REPORTED_BROKEN = "There is no open refrigerator malfunction reported at your facility."
+
+    FRIDGE_CONFIRM_PRODUCTS_COLLECTED_FROM = ("Please reply with 'rc' to confirm you have collected your EPI "
+        "products from %(facility)s")
+
+    FRIDGE_CONFIRM_PRODUCTS_COLLECTED = "Please reply with 'rc' to confirm you have collected your EPI products"
+
+    FRIDGE_NOT_REPORTED_FIXED = ("Your refrigerator has not been reported as fixed yet. If it has been "
+        "fixed, please respond with 'rf'")
+
+    FRIDGE_ALREADY_CONFIRMED_COLLECTED = "You have already confirmed that you have collected your EPI products."
+
+    FRIDGE_CONFIRMATION_RESPONSE = "Thank you for confirming that you have collected your EPI products."
```

Comment:
at some point we should just get input on the exact words of all these messages but no need to change anything for now.